### DISCUSSION TITLE
Delayed status effect that can't find target position should not be executed

### DISCUSSION THREAD

**Discussion by A2ur3** [FakeFishGames/Barotrauma#15284] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [X] My issue happened while using mods.

### What happened?

At present a delayed status effect will still happen in 0 coordinate position if its target have already been removed
for example a creature will spawn 2 husks after 10 seconds when it is killed. but for some reason it got removed (despawnnow? eaten by other creature?)
when delay time's up the status effect still spawn husks but at origin position

### Reproduction steps

.

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Happens in both single player and multiplayer

### Version

v1.6.19.1 (Unto the Breach Update Hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by A2ur3** [comment]
#14975

**Comment by Regalis11** [comment]
Thank you for the report! This has been addressed in the current unstable build, fix coming up to the public version in a future update.
